help/en/html/hardware/marklin/index.shtml

```diff
@@ -12,7 +12,12 @@
     <!--#include virtual="../SidebarUp.shtml" -->
 
     <div id="mainContent">
-      <h1>Hardware Support: M&auml;rklin Command Station 2 (CS2)</h1>
+      <h1>Hardware Support: M&auml;rklin </h1>
+      JMRI supports two M&auml;rklin command stations:
+      the <a href="#cs2">Command Station 2 (CS2)</a> and
+      the <a href="">CdB CC-Schnitte 2</a>.
```

Comment:
@bobjacobsen 
Issue https://github.com/JMRI/JMRI/issues/11653 has some information about it.

Quote:
The CC-Schnitte is a third party USB interface for the Marklin CAN Bus. It is usually directly attached to the 60113 Digital Connector Box, with the MobileStation 2 attached to the second port. In the Marklin system, the Connector Box receives commands from devices attached to the CAN Bus. It acts as a mini booster, but also translates them into a proper DCC/mm2/mfx signal. CC-Schnitte combines with the Connector Box in a fashion similar as the already supported CentralStation 2.

I suggest that this text is added to this page.